Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -337,7 +337,6 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     /// Executes the provided action on the message
     open func handleTapOnAttachmentAction(
         _ action: AttachmentAction,
-        for attachment: ChatMessageAttachment,
```

Comment:
maybe we should replace it with `AttachmentId`?

Reply:
This was changed in the PR for `GiphyAttachmentViewInjector` but you are right 👍 I'd add `attachment: ChatMessageGiphyAttachment` param to delegate's method (now only `action` and `indexPath` are given)

Reply:
cc @DominikBucher12